lxqt-leave/main.cpp

```diff
@@ -106,6 +106,6 @@ int main(int argc, char *argv[])
                 Qt::AlignCenter,
                 dialog.size(),
                 qApp->desktop()->screenGeometry(QCursor::pos())));
-    dialog.setMaximumSize(dialog.minimumSize());
+    dialog.setMaximumSize(dialog.size());
```

Comment:
Yeah, I thought that. And tested. `sizeHint()` returned a smaller size than `size()`. As you can see, `size()` has been used in the previous function call. We can set `minimumSize` to `sizeHint()` and rest assured, do you agree?